### PR TITLE
Custom signals

### PR DESCRIPTION
**Pull request by lisandor**
- Added custom Signal (post_update) to get triggered after a model update with m2m relationships updated. 
- Added a new Mixin to set old_instance attribute in a model.
- Added a new serializer to extend from a django serializer that adds the new post_update signal

### CODE REVIEW

**Review by mjroson — APPROVED** [review]
+1

File: signalhooks/signals.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from django import dispatch
+
+pre_update = dispatch.Signal(providing_args=["Instance"])
```
Comment (gsimoncini) [inline signalhooks/signals.py]: volemos esta

**Review by gsimoncini — COMMENTED** [review]
+1 luego de borran la signal pre_update

**Review by juanmfite — COMMENTED** [review]
+1

**Pull request merged**